lib/smart_proxy_remote_execution_ssh.rb

```diff
@@ -5,7 +5,7 @@
 module Proxy::RemoteExecution
```

Comment:
this needs to be broken to
```
module Proxy
  module RemoteExecution
```
because if we run it standalone, Proxy does not exist